components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -371,9 +372,7 @@ protected void triggerNotification(Event event, String userName, UserStoreManage
         try {
             AccountServiceDataHolder.getInstance().getIdentityEventService().handleEvent(identityMgtEvent);
         } catch (IdentityEventException e) {
-            String errorMsg = "Error occurred while calling triggerNotification, detail : " + e.getMessage();
-            log.error(errorMsg);
-            //We are not throwing any exception from here, because this event notification should not break the main flow.
+            throw new AccountLockException("Error occurred while sending notification", e);
```

Comment:
According to the comment on previous code, this should not be thrown. It should be logged instead. @IsuraD can you please review and confirm?